src/frontend/src/lib/i18n/en.json

```diff
@@ -48,6 +48,8 @@
 		},
 		"error": {
 			"no_internet_identity": "No internet identity.",
+			"invalid_pouh_credential": "Sorry, but there was an error while validating the Humanity Credential. Please contact the issuer and request the credential again.",
```

Comment:
curiosity question: "Humanity Credential" is a name therefore the capital letters?

Reply:
That's how I see it so far. In the settings, the key is called "Humanity Credential", and this messages refer to that line in settings.

Reply:
Good to know.